nengo_fpga/networks/fpga_pes_ensemble_network.py

```diff
@@ -498,11 +498,17 @@ def connect(self):
                 "Did not receive connection from board within "
                 + "specified timeout (%fs)." % self.connect_timeout
             )
+
         if self.recv_buffer[0] < 0.0:
             # Received a "terminate client" packet from the board, terminate the
             # Nengo simulation.
+            reason = ""
+            if self.recv_buffer[0] <= -10:
+                reason = "Unable to acquire FPGA resource!"
+            elif self.recv_buffer[0] <= -20:
```

Comment:
the two conditions should be reversed. If the board returns -20, it will hit (and pass) `< -10` first.

Reply:
good call... Friday afternoon caught me not thinking XD